oneclick/routes.py

```diff
@@ -0,0 +1,110 @@
+from transbank.error.transbank_error import TransbankError
+from transbank.oneclick.request import MallTransactionAuthorizeDetails
+
+from oneclick import bp
+from flask import render_template, request
+from transbank.oneclick.mall_inscription import MallInscription
+from transbank.oneclick.mall_transaction import MallTransaction
+import random
+
+
+@bp.route('start', methods=['GET'])
+def show_start():
+    return render_template('/oneclick/start.html')
+
+
+@bp.route('status', methods=['GET'])
+def show_status():
+    return render_template('/oneclick/status_form.html')
+
+
+@bp.route('start', methods=['POST'])
+def start():
+    user_name = request.form.get('user_name')
+    email = request.form.get('email')
+    response_url = request.form.get('response_url')
+
+    resp = MallInscription.start(
+        user_name=user_name,
+        email=email,
+        response_url=response_url)
+
+    return render_template('oneclick/started.html', resp=resp, req=request.form)
+
+
+@bp.route('finish', methods=['POST'])
+def finish():
+    req = request.form
+    token = request.form.get('TBK_TOKEN')
+
+    resp = MallInscription.finish(token=token)
+    buy_order_1 = str(random.randrange(1000000, 99999999))
```

Comment:
I would change these to be taken from the form, so the user can change them, so they can use this to test integration with their credentials and a baseline implementation